README.md

```diff
@@ -144,10 +144,10 @@ FreeBSD Notes
 -------------
 You need to compile using gmake - regular make only appears to work, but doesn't in fact. Use gmake, not make.
 
-The clang compiler installed through FreeBSD's package manager does not expose all of the C++11 features needed under `std=gnuc++11`. Force the compiler to use `std=c++11` mode instead.
+The clang compiler installed through FreeBSD's package manager does not expose all of the C++17 features needed under `std=gnuc++11`. Force the compiler to use `std=c++17` mode instead.
```

Comment:
If that's still true:
```suggestion
The clang compiler installed through FreeBSD's package manager does not expose all of the C++17 features needed under `std=gnuc++17`. Force the compiler to use `std=c++17` mode instead.
```

Reply:
Default on FreeBSD is `gnuc++14`, see https://www.freebsd.org/cgi/man.cgi?query=cc&apropos=0&sektion=0&manpath=FreeBSD+13.0-RELEASE+and+Ports&arch=default&format=html

So it should become:

...C++17 features  needed under the default `std=gnuc++14`